src/routes/groups/groups.controller.js

```diff
@@ -0,0 +1,60 @@
+const groupsData = require('./../../../db/groups.data.json')
+const fs = require("fs");
+const { promisify } = require("util");
+
+const writeFile = promisify(fs.writeFile);
+
+const listGroup = (req, res) => {
+  return res.json(groupsData);
+};
+
+const postGroup = async (req, res) => {
+  const id = groupsData.groups.length + 1;
+  const newgroupsData = {
+    groups: [...groupsData.groups, { id: parseInt(id, 10), ...req.body }]
+  };
+  await writeFile("db/groups.data.json", JSON.stringify(newgroupsData));
+  res.status(201);
```

Comment:
Good job, using the correct 'created' status code!  You might consider putting success status codes in an object the same way you did with the errors too.